requirements/production.txt

```diff
@@ -2,5 +2,5 @@ docutils>=0.12
 Jinja2>=2.7.3
 MarkupSafe>=0.23
 Pygments>=1.6
-Sphinx>=3.0,<3.6
+Sphinx>=3.0,<5
```

Comment:
This also needs to be changed in the `setup.py`, I'll fixup and push to branch in a sec to save some hassle.

Reply:
I'll also bump minimum python requirement for Breathe to 3.6 to match the changing Sphinx support.